package.json

```diff
@@ -70,7 +70,7 @@
     "rollup-plugin-url": "^1.4.0",
     "sass-loader": "^8.0.0",
     "style-loader": "^1.0.0",
-    "webpack": "^4.41.2"
+    "webpack": "^4.41.0"
```

Comment:
Had to reduce the version of `webpack` in order for the latest stable version of `react-scripts` to pass a compiling check that was breaking the build in the example app